.peoplemap

```diff
@@ -28,6 +28,7 @@ Florian Haas <florian@citynetwork.eu> Florian Haas <florian@hastexo.com>
 Greg Farnum <gfarnum@redhat.com> Greg Farnum <greg@inktank.com>
 Gregory Meno <gmeno@redhat.com> <gregory.meno@inktank.com>
 Guillaume Abrioux <gabrioux@ibm.com> <gabrioux@redhat.com>
+Abhishek Kane <abhishek.kane@ibm.com> <abhishek.kane@gmail.com>
```

Comment:
When taking a second look at this, it looks like both the `.peoplemap` and `.mailmap` are ordered alphabetically by first name, so we should probably move your name up the list to a spot where that holds in both places

Reply:
I have made all changes in ordered fashion. Sorry, i didn't notice it earlier.